Makefile

```diff
@@ -150,6 +139,39 @@ clean:
 distclean: clean
 	rm -rf vendor/
 
+###############################################################################
+###                                Release                                  ###
+###############################################################################
+
+# create tag and run goreleaser without publishing
+create-release-dry-run:
+ifneq ($(strip $(TAG)),)
+	@echo "--> Dry running release for tag: $(TAG)"
+	@echo "--> Create tag: $(TAG) dry run"
+	git tag -s $(TAG) -m $(TAG)
+	git push origin $(TAG) --dry-run
+	@echo "--> Delete local tag: $(TAG)"
+	@git tag -d $(TAG)
+	@echo "--> Running goreleaser"
+	TM_VERSION=$(TM_VERSION) goreleaser release --snapshot --clean
```

Comment:
Gorealaser has to be installed before running it locally using sthg like:
`@go install github.com/goreleaser/goreleaser@latest`